src/rpc/blockchain.cpp

```diff
@@ -2014,7 +2014,7 @@ bool FindScriptPubKey(std::atomic<int>& scan_progress, const std::atomic<bool>&
         }
         if (count % 256 == 0) {
             // update progress reference every 256 item
-            uint32_t high = 0x100 * *key.hash.begin() + *(key.hash.begin() + 1);
+            uint32_t high = 0x100 * *UCharCast(key.hash.begin()) + *(UCharCast(key.hash.begin()) + 1);
```

Comment:
This seems unrelated?

Reply:
`Txid::begin` etc. return `std::byte*` which can't be used for arithmetic operations.

See https://en.cppreference.com/w/cpp/types/byte.

Reply:
I was confused where `Txid` comes to play here. But I see `key` is a `COutPoint`.